### ISSUE TITLE
Grid and gizmos are counted in the information panel in 3D editor

### ISSUE THREAD

**Issue by timothyqiu** [godotengine/godot#49820]
### Godot version

3.3.2.stable

### System information

macOS 11.4

### Issue description

Grid and gizmos are counted in various metrics in the "View Information" panel.

I understand that grid and gizmos are part of the scene in the editor, and may be hard / impossible to separate. But I think it's unexpected and a bit misleading at first glance.

Empty scene:
![empty](https://user-images.githubusercontent.com/372476/122846383-4f7aa880-d338-11eb-8ca3-711060f24f19.png)

Huge boost in vertices when the root node is selected in the empty scene:
![selected](https://user-images.githubusercontent.com/372476/122846388-50abd580-d338-11eb-9caa-cfe6160f39f2.png)

### Steps to reproduce

1. Create a 3D scene
2. Turn on the "View Information" panel

**Comment by Calinou** [comment]
This was reported in the comments of this issue: https://github.com/godotengine/godot/issues/23826

I'm fairly sure this is a duplicate of another issue, but I can't find it right now.

Also, it may not be possible to fix this without moving gizmo rendering to a separate layer (so that it can also ignore things like glow and depth of field).

**Comment by Zireael07** [comment]
Yep, I'm almost certain I (or someone else) raised the problem somewhere else, too...

At any rate, as mentioned in #23826, this needless counting is making it very difficult to figure out which meshes are the costliest in terms of drawcalls/vertices. As a stopgap, it should be possible to calculate the amount that grid/gizmo use and just subtract it from the reported number.

**Comment by Calinou** [comment]
> As a stopgap, it should be possible to calculate the amount that grid/gizmo use and just subtract it from the reported number.

The issue is that this number can change easily at run-time, and recalculating it all the time can get tedious.

**Comment by Chaosus** [comment]
Then some kind of tag should be attached to the mesh instance to indicate that mesh should not be considered? Or rendering server should have a list of excluded mesh RID's.

**Comment by fire** [comment]
I have a per object rid prototype, but I only work on master.

https://github.com/godotengine/godot/pull/43830

**Comment by Calinou** [comment]
I have a first draft of a property that can be used to ignore specific objects in the reported metrics: https://github.com/Calinou/godot/tree/renderingserver-add-ignore-in-render-info

There are many outstanding issues that would need to be fixed before this is usable though (see the commit message in the above branch).
